ositrace/ositrace_shacl.ttl

```diff
@@ -0,0 +1,286 @@
+            sh:name "startTime"@en ;
+            sh:order 6 ;
+            sh:path ositrace:startTime ],
+        [ skos:example "2024-05-11T13:20:05.250120Z" ;
+            sh:datatype xsd:dateTimeStamp ;
+            sh:description "Exact stop timestamp of the recorded trace"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of stopTime failed!"@en ;
+            sh:minCount 1 ;
+            sh:name "stopTime"@en ;
+            sh:order 7 ;
+            sh:path ositrace:stopTime ],
+        [ sh:description "Host moving object in trace file"@en ;
+            sh:maxCount 1 ;
+            sh:name "hostMovingObject"@en ;
+            sh:node ositrace:MovingObjectShape ;
+            sh:order 8 ;
+            sh:path ositrace:hostMovingObject ],
+        [ sh:description "Target moving object in trace file"@en ;
+            sh:name "targetMovingObject"@en ;
+            sh:node ositrace:MovingObjectShape ;
+            sh:order 9 ;
+            sh:path ositrace:targetMovingObject ],
+        [ sh:description "Description of event of interest in trace file"@en ;
+            sh:name "event"@en ;
+            sh:node ositrace:EventShape ;
+            sh:order 10 ;
+            sh:path ositrace:event ] ;
+    sh:targetClass ositrace:Content .
+
+ositrace:DataSourceShape a sh:NodeShape ;
+    sh:property [ skos:example "PMSF DroneTracker; ARS 548 Sensor Output" ;
+            sh:datatype xsd:string ;
+            sh:description "Main acquisition device"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of measurementSystem failed!"@en ;
+            sh:name "measurementSystem"@en ;
+            sh:order 1 ;
+            sh:path ositrace:measurementSystem ],
+        [ skos:example "scanner, camera, osm network, aerial images, ..." ;
+            sh:datatype xsd:string ;
+            sh:description "Basic data for the creation of the trace "@en ;
+            sh:message "Validation of usedDataSources failed!"@en ;
+            sh:name "usedDataSources"@en ;
+            sh:order 0 ;
+            sh:path ositrace:usedDataSources ] ;
+    sh:targetClass ositrace:DataSource .
+
+ositrace:FormatShape a sh:NodeShape ;
+    sh:property [ skos:example "3.7.0" ;
+            sh:datatype xsd:string ;
+            sh:description "Version of data format"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of version failed!"@en ;
+            sh:name "version"@en ;
+            sh:order 1 ;
+            sh:path ositrace:version ],
+        [ skos:example "ASAM OSI GroundTruth; ASAM OSI SensorView; ASAM OSI SensorData" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:in ("ASAM OSI GroundTruth" "ASAM OSI SensorView" "ASAM OSI SensorData") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path ositrace:type ] ;
```

Comment:
Maybe `sh:path ositrace:rootLevelType` or `sh:path ositrace:rootLevelMessageType`?

Reply:
Do we envision something else that is going to be an ositrace type? Otherwise I'd keep it simple.

Reply:
I first thought about the "*.osi", and "*.txth" serialization but this would be another attribute called "serialization". So its fine if you think we should keep it simple. I'm a fan of more precise keywords.